core/src/main/resources/hudson/model/View/sidepanel.jelly

```diff
@@ -51,7 +51,7 @@ THE SOFTWARE.
           <l:task href="${rootURL}/${it.viewUrl}asynchPeople/" icon="icon-user icon-md" title="${%People}"/>
         </j:otherwise>
       </j:choose>
-      <l:task href="${rootURL}/${it.viewUrl}builds" icon="icon-notepad icon-md" title="${%Build History}"/>
+      <l:task href="${rootURL}/${it.viewUrl}builds" icon="symbol-build-history" title="${%Build History}"/>
```

Comment:
This invokes the "Time Machine" UI or what is this icon?

Reply:
Aha it definitely does,. What do you think of this one/do you have a better icon in mind?

<img width="304" alt="image" src="https://user-images.githubusercontent.com/43062514/156634735-93425aff-9316-4d9a-af45-ba86909842a2.png">

Reply:
That has more of a "record" feel, so probably? Time Machine seems too clever, and "going back in time" is probably more for https://plugins.jenkins.io/jobConfigHistory/ than build history.